Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        public struct Person
+        {
+            public Person(string name, string password)
+            {
+                Name = name;
+                Password = password;
+            }
+
+            public string Name { get; set; }
+            public string Password { get; set; }
+        }
+
+        internal static void Demo()
+        {
+            Console.WriteLine("Would you like to register or login");
+            Console.WriteLine("Choose R voor register");
+            Console.WriteLine("Choose L voor login");
+            var choice = Console.ReadLine();
+            switch (choice)
+            {
+                case "R":
```

Comment:
This is very bad, because you have magic hardcoded strings. Please use constants at least or ideally enum.

Reply:
You mean something like this 

```
Enum Menu
{
    Register 
    Login 
} 
``` 

and use that in the switch statement. That means that the user has to type in Register of Login if I see it right.

Reply:
hmm, 

When I changed it to : 

```
internal static void Demo()
        {
            Console.WriteLine("Would you like to register or login");
            Console.WriteLine("Choose R for register");
            Console.WriteLine("Choose L for login");
            var choice = Console.ReadLine();
            switch (choice)
            {
                case Menu.MenuChoices.R:
                    Register();
                    break;

                case Menu.MenuChoices.L:
                    Login();
                    break;

                default:
                    Console.WriteLine("This is not a right choice. Only R and L are valid");
                    break;
            }
        }
 ``` 

I get a error that a Enum cannot be converted to a string. 
Any hint how to solve this one ?

Reply:
Enum.Parse(typeof(yourEnumType), input);

Reply:
thanks. that worked. Never thought of the typeof  we learned at this lesson